trackomatic.js

```diff
@@ -23,24 +48,28 @@ function providePlugin(pluginName, pluginConstructor) {
 }
 
 // Plugin constructor.
-//Everything takes place inside this.
+// Everything takes place inside this.
 function Trackomatic(tracker, config) {
-
   //Check for the trackomatic object globally; instantiate a local one if it's not set.
   window._trackomatic = window._trackomatic || {};
 
   // Sanity check
-  _trackomatic.config  = typeof config !== "undefined" ? config : {};
-  console.log("Loaded trackomatic on tracker " + tracker.get('name') + " with the config object " + JSON.stringify(config));
+  _trackomatic.config = typeof config !== 'undefined' ? config : {};
+
+  console.log(
+    'Loaded trackomatic on tracker ' + tracker.get('name') + \
+    ' with the config object ' + JSON.stringify(config)
+  );
 
 
   // Javascript error tracking with message and line number
   // Should use exception tracking in next version: https://developers.google.com/analytics/devguides/collection/analyticsjs/exceptions
   // Would also be worth sampling + throttling
   var oldonerror = window.onerror;
+
   window.onerror = function(msg, url, line) {
     if (oldonerror) {
-      oldonerror.apply(this, arguments );
+      oldonerror.apply(this, arguments);
```

Comment:
It should be loaded pretty late, since it's loaded as a part of GA, and the loaded again.

`window.onerror` seems to be the only option for IE8 and below, so we need to talk about standardizing on our support.